src/store/metadata/stub/content/editions/cpih_2022-01.json

```diff
@@ -10,26 +10,27 @@
       "in_series": "https://staging.idpd.uk/datasets/cpih",
       "identifier": "2022-01",
       "title": "Consumer Prices Index including owner occupiers' housing costs (CPIH)",
-      "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a...",
-      "description": "The Consumer Prices Index...",
+      "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH).",
```

Comment:
Changes to summary and description are great, but they should still differ from the title to some extent.